testing/python/fixture.py

```diff
@@ -417,6 +417,7 @@ def test_lookup_error(unknown):
             "*1 error*",
         ])
         assert "INTERNAL" not in result.stdout.str()
+        assert 'monkeypatch:session' not in result.stdout.str()
```

Comment:
I would change it no problem, but I agree it is really minor and not worth having to restart CI all over again. :grin: